python/sqlflow_submitter/xgboost/pai_rabit.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import http.client
+import socket
+import threading
+import time
+from contextlib import closing
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+from sqlflow_submitter.xgboost.tracker import RabitTracker
```

Comment:
Can we import `RabitTracker` directly from xgboost installation? so that we can avoid copy that file.

Reply:
Good point!

Reply:
It looks like we can't use the `xgboost.RabitTracker` directly. Because of:
1. The `RabitTracker` only contained in `xgboost>=1.0.0`
1. `xgboost>=1.0.0` only supported by Python3.
1. PAI tf1.15 only supports Python2
We are using PAI tf1.15, so I add this file back. #2200